scripts/tls-common.sh

```diff
@@ -2,23 +2,29 @@
 
 export IRONIC_CERT_FILE=/certs/ironic/tls.crt
 export IRONIC_KEY_FILE=/certs/ironic/tls.key
-export IRONIC_CACERT_FILE=/certs/ca/ironic/tls.crt
 export IRONIC_INSECURE=${IRONIC_INSECURE:-false}
 export IRONIC_SSL_PROTOCOL=${IRONIC_SSL_PROTOCOL:-"-ALL +TLSv1.2 +TLSv1.3"}
 export IPXE_SSL_PROTOCOL=${IPXE_SSL_PROTOCOL:-"-ALL +TLSv1.2 +TLSv1.3"}
+export HTTPD_SSL_PROTOCOL=${HTTPD_SSL_PROTOCOL:-"-ALL +TLSv1.2 +TLSv1.3"}
```

Comment:
General comment on all variables and paths: could it be something more descriptive than "httpd"? Everything here uses httpd in one way or another. IMAGE_SERVER or something maybe?

Reply:
Done.